Explorer/Assets/DCL/AvatarRendering/Wearables/Helpers/WearableComponentsUtils.cs

```diff
@@ -63,11 +77,41 @@ public static GetWearablesByPointersIntention CreateGetWearablesByPointersIntent
                 return null;
             }
 
+            if (attachment.ManifestResult?.Asset == null)
+            {
+                try
+                {
+                    attachment.ManifestResult  = new StreamableLoadingResult<SceneAssetBundleManifest>(
+                        await LoadWearableAssetBundleManifestUtils.LoadWearableAssetBundleManifestAsync(assetBundleURL, attachment.GetHash(), ReportCategory.WEARABLE, cancellationTokenSource?.Token ?? new CancellationToken()));
+                }
+                catch (Exception e)
+                {
+                    ReportHub.Log(ReportCategory.WEARABLE, $"Wearable {attachment.GetHash()} doesnt have a manifest");
+                    CreateWearableThumbnailPromiseTexture(realmData, thumbnailPath, attachment, world, partitionComponent, cancellationTokenSource);
+                    return null;
```

Comment:
I think is not cohesive to return `null` because the thumbnail promise has been created, but not as asset bundle. Perhaps you could return an object that is composed either with an asset bundle promise or a thumbnail texture promise. WDYT?

Reply:
Changed the logic of the contract make it cohesive. 
Since this is the only [place](https://github.com/decentraland/unity-explorer/blob/c7d564a967c438d1c6d065dc1dc36cdc17a1fa13/Explorer/Assets/DCL/AvatarRendering/Wearables/ECSThumbnailProvider.cs#L51) where we are using the new method, there isnt a real reason to return the promise. So I changed the return type to null.